websites/R/RTLplay/presence.ts

```diff
@@ -0,0 +1,555 @@
+	Deferred = "https://i.imgur.com/o69hUKD.gif",
+	LiveAnimated = "https://i.imgur.com/6oezfP6.gif",
+	Listening = "https://i.imgur.com/9ZFChOG.png",
+	RTLPlay = "https://i.imgur.com/1f5rMxV.png",
+	RTLTVi = "https://i.imgur.com/wnjbhCe.png",
+	RTLClub = "https://i.imgur.com/8FwZa7m.png",
+	RTLPlug = "https://i.imgur.com/jfYLcJu.png",
+	BelRTL = "https://i.imgur.com/1yZelPm.png",
+	Contact = "https://i.imgur.com/ULP7pgr.png",
+}
+
+function getChannel(channel: string) {
+	switch (true) {
+		case channel.includes("tvi"): {
+			return {
+				channel: "RTL TVi",
+				type: ActivityType.Watching,
+				logo: Assets.RTLTVi,
+			};
+		}
+		case channel.includes("club"): {
+			return {
+				channel: "RTL club",
+				type: ActivityType.Watching,
+				logo: Assets.RTLClub,
+			};
+		}
+		case channel.includes("plug"): {
+			return {
+				channel: "RTL plug",
+				type: ActivityType.Watching,
+				logo: Assets.RTLPlug,
+			};
+		}
+		case channel.includes("bel"): {
+			return {
+				channel: "Bel RTL",
+				type: ActivityType.Listening,
+				logo: Assets.BelRTL,
+			};
+		}
+		case channel.includes("contact"): {
+			return {
+				channel: "Radio Contact",
+				type: ActivityType.Listening,
+				logo: Assets.Contact,
+			};
+		}
+		default: {
+			return {
+				channel,
+				type: ActivityType.Watching,
+				logo: Assets.RTLPlay,
+			};
+		}
+	}
+}
+
+function exist(selector: string) {
+	return document.querySelector(selector) !== null;
+}
+
+const shortenedURLs: Record<string, string> = {};
+async function getShortURL(url: string) {
+	if (!url || url.length < 256) return url;
+	if (shortenedURLs[url]) return shortenedURLs[url];
+	try {
+		const pdURL = await (
+			await fetch(`https://pd.premid.app/create/${url}`)
+		).text();
+		shortenedURLs[url] = pdURL;
+		return pdURL;
+	} catch (err) {
+		presence.error(err);
+		return url;
+	}
+}
+
+presence.on("UpdateData", async () => {
+	const { hostname, href, pathname } = document.location,
+		presenceData: PresenceData = {
+			name: "RTLplay",
+			largeImageKey:
+				hostname === "www.radiocontact.be" ? Assets.Contact : Assets.Animated, // Default
+			largeImageText: "RTLplay",
+			type: ActivityType.Watching,
+		},
+		[lang, usePresenceName, useChannelName, privacy, time, buttons, poster] =
+			await Promise.all([
+				presence.getSetting<string>("lang").catch(() => "en"),
+				presence.getSetting<boolean>("usePresenceName"),
+				presence.getSetting<boolean>("useChannelName"),
+				presence.getSetting<boolean>("privacy"),
+				presence.getSetting<boolean>("timestamp"),
+				presence.getSetting<number>("buttons"),
+				presence.getSetting<boolean>("usePosterImage"),
+			]);
+
+	if (oldLang !== lang) {
+		oldLang = lang;
+		strings = await getStrings();
+	}
+
+	switch (true) {
+		/* MAIN PAGE (Page principale)
+
+		(https://www.rtlplay.be/) */
+		case pathname === "/" ||
+			(pathname.split("/")[1] === "rtlplay" &&
+				pathname.split("/").length <= 2): {
+			presenceData.details = (await strings).browsing;
```

Comment:
Now u can change the await (strings). to strings.
```suggestion
			presenceData.details = strings.browsing;
```